style.css

```diff
@@ -1 +1,291 @@
-/*  Add your styling here */
+@font-face {
+  font-family: Questrial-Regular;
+  src: url(img/OFL.txt);
+}
+
+*:root {
+  --primary: #f2f5f8;
+  --secondary: #f56ad7;
+}
+
+body,
+a,
+ul,
+li,
+p {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  font-family: "Raleway";
+  font-size: 22px;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+.homepage-main {
+  overflow-x: hidden;
+}
+
+li {
+  list-style-type: none;
+  display: inline;
+  padding: 20px;
+  justify-content: end;
+}
+
+ul a {
+  text-decoration-color: none;
+}
+
+.button {
+  background: var(--secondary);
+  border: 2px solid var(--secondary);
+  border-radius: 2px;
+  color: var(--primary);
+  padding: 0.5em;
+}
+
+.button:hover {
+  background-color: whitesmoke;
+  color: var(--secondary);
+}
+
+.btn-tab {
+  margin: 1em;
+}
+
+.message,
+image {
+  grid-column: 3/4;
+}
+
+/*mobile*/
+.homepage-main {
+  overflow-x: hidden;
+  height: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: auto auto auto auto;
+  grid-template-areas:
+    "hero     hero    hero     hero"
+    "feature  feature feature  feature"
+    "product  product product  product"
+    "aside    aside   aside     aside";
+}
+
+.homepage-hero {
+  grid-column: 1/5;
+  grid-area: hero;
+  background-color: turquoise;
+}
+
+h1 {
+  font-family: Questrial-Regular;
+}
+
+.h1header {
+  margin: auto auto 0 12em;
+  text-align: right;
+  padding-right: 1em;
+  font-family: cursive;
+}
+
+.cake {
+  text-align: justify;
+  margin: 20px 10px 10px 0px;
+}
+
+.homepage-feature {
+  grid-area: feature;
+  grid-column: 1/5;
+  padding: 1em;
+}
+
+svg {
+  padding-left: 21em;
+  margin-top: 2em;
+}
+
+.homepage-products {
+  display: grid;
+  grid-area: product;
+  grid-row: 3;
+  grid-gap: 10px;
+  grid-auto-rows: auto;
+  grid-auto-flow: dense;
+  margin: 0.5em;
+  padding: auto 0.5em;
+}
+
+.homepage-aside {
+  grid-area: aside;
+  display: inline-grid;
+  margin-top: 1.5em;
+  margin: 5em;
+}
+
+.site-nav {
+  display: none;
+}
+
+.featureimg {
+  display: grid;
+  grid-column-start: 5;
+  grid-column-end: 7;
+  grid-row: 2;
+  width: 100%;
+  margin: 1.5em 0 0 0;
+}
+
+.p-logo {
+  margin: -50px;
+  padding-left: 110px;
+  font-weight: bold;
+}
+
+.image {
+  margin-top: 1em;
+  padding-left: 1em;
+}
+
+.social {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+}
+
+.footer {
+  padding: 10px;
+  margin-top: -1em;
+}
+
+.svg-social {
+  padding-left: 10em;
+}
+
+/* TABLET: */
+@media screen and (min-width: 614px) {
```

Comment:
I believe the homework asked only two breakpoints: 540px and 900px, therefore 614px was not requested